service/pom.xml

```diff
@@ -79,23 +85,16 @@
         </dependency>
 
         <dependency>
-            <groupId>com.h2database</groupId>
-            <artifactId>h2</artifactId>
-            <scope>test</scope>
-            <version>${h2-database.version}</version>
-        </dependency>
-
-        <dependency>
             <groupId>com.zaxxer</groupId>
             <artifactId>HikariCP</artifactId>
             <version>2.3.9</version>
```

Comment:
What I think we should do is just plan on doing a monthly or bi monthly check on dependencies and upgrade them if we can.  Unless there is something in the upgarde